test/expectations/selection_ranges/heredoc.exp.json

```diff
@@ -25,7 +25,7 @@
                     },
                     "end": {
                         "line": 2,
-                        "character": 0
+                        "character": 8
```

Comment:
Do we know which change leads to this difference?

Reply:
I can't actually find a commit/PR about this, perhaps it's a side-effect of some other change. The new position is the correct though, since we want it to include to the end of `HEREDOC` in https://github.com/Shopify/ruby-lsp/blob/main/test/fixtures/heredoc.rb